packages/web/docs/src/content/gateway/other-features/security/aws-sigv4.mdx

```diff
@@ -0,0 +1,103 @@
+---
+searchable: false
+---
+
+# AWS Signature Version 4 (SigV4)
```

Comment:
maybe add a simple mermaid chart here to make it more clean on what/how it does?

Reply:
Done!